### ISSUE TITLE
VS Code can't enable extension

### ISSUE THREAD

**Issue by Aukstkalnis** [kube/vscode-clang-complete#15]
OS: Windows 7 x64
VSC: 1.8.1

**What did you do?**
Downloaded extension via explorer and reset VS Code.

**What did you expected?**
To see that extension works.

**What was the result?**
Every time when I reload VS Code I can only see that extension asks for reload and is inactive.

**Comment by kube** [comment]
Hi, I'm sorry for this.

Last update seems to have broken some stuff.

The thing is, I'm working on a whole new version, that will work differently and implement more features and fix all the bugs from this version, but will be implemented in a very different way.

So I don't take time to fix this current version.

You can watch the repo if you want to see when I'll push the new version.

**Comment by Aukstkalnis** [comment]
Cool :) I am looking forward to your new release :) Thanks for your great work :)

**Comment by agauniyal** [comment]
Has the new version released?

**Comment by assumptionsoup** [comment]
@agauniyal  No.  There have not been any updates on this project since Dec 18th.  As you can see from the main page...

**Comment by kube** [comment]
I have a lot of delay on my current project, and I cannot take too much time to work on this for now.

I started the principal (and separated) module of the future release, which is a wrapper around Clang, permitting to easily handle the AST of a source file.

I won't give up this project, I just don't do C on my current project, but I will on the next one, and this extension will be part of my workflow, so I will take time to optimize it.

If you are interested in the future release of this extension, please watch the repo.

**Comment by agauniyal** [comment]
Sure, take your time and we'll watch the repo for changes.
